### ISSUE TITLE
Drag and drop images into notebook cells from left file browser pane.

### ISSUE THREAD

**Issue by akisaini** [jupyterlab/jupyterlab#15911]
In JupyterLab, I can drag and drop an image into a markdown cell, and it shows up as an attached image.
Would it be possible to have this Drag'n'Drop feature for notebook cells as well?
Current behavior shows drop areas to open the file.

Below comment in another issue briefly mentions the behavior:
_Originally posted by @krassowski in https://github.com/jupyterlab/jupyterlab/issues/15615#issuecomment-1883287220_

**Comment by krassowski** [comment]
> In JupyterLab, I can drag and drop an image into a markdown cell, and it shows up as an attached image.
> Would it be possible to have this Drag'n'Drop feature for notebook cells as well?

Do you mean into code cells? Code cells have source and outputs, while markdown cells have attachments. What would dropping an image into a code cell mean?

**Comment by akisaini** [comment]
> > In JupyterLab, I can drag and drop an image into a markdown cell, and it shows up as an attached image.
> > Would it be possible to have this Drag'n'Drop feature for notebook cells as well?
> 
> Do you mean into code cells? Code cells have source and outputs, while markdown cells have attachments. What would dropping an image into a code cell mean?

Yes, into the code cells. I am essentially trying to update the source of a cell by dragging and dropping an image into it. 

More elaborately, I am trying for It to update the path of a parameter, if possible. 
`function(openImage = 'provide/path/to/image')`
So, instead of the user inputting the path, the goal would be to drag and drop an image from the left file browser pane and add/update the path to an image in the `openImage` parameter.

**Comment by JasonWeill** [comment]
Expected behavior: Dragging a file from the left panel into a code cell would cause the file's path to appear in the code cell.

**Comment by akisaini** [comment]
@krassowski @JasonWeill 
> > In JupyterLab, I can drag and drop an image into a markdown cell, and it shows up as an attached image.
> > Would it be possible to have this Drag'n'Drop feature for notebook cells as well?
> 
> Do you mean into code cells? Code cells have source and outputs, while markdown cells have attachments. What would dropping an image into a code cell mean?

My original question was not clearly put together. My goal here is to drag and drop an image from the file browser on the left pane into the output of a code cell, which would then update the same output.
